### DISCUSSION TITLE
resizing cluster

### DISCUSSION THREAD

**Discussion by KSPlatform** [siderolabs/talos#6721] [General]
Hi,
Will you please guide me for increasizing the number of worker nodes for the Talos cluster?
Thanks

**Comment by smira** [comment]
That is actually the easiest thing with Talos - just boot more nodes with the machine config for workers (`worker.yaml`). New nodes will join the cluster automatically when booted.

**Comment by KSPlatform** [comment]
Hi, for VSphere users, added:
disk.enableUUID: TRUE
guestinfo.talos.config: base64_of_worker_yaml in advanced settings for the new clone..
Thanks
